classes/04class/exercises/c04-s301/carineoliveira/cli_commands.txt

```diff
@@ -0,0 +1,30 @@
+1. Create a new bucket with name <your_github_username> 
+
+carineoliveira@Carines-MBP carineoliveira % aws s3api create-bucket --bucket carineoliveira --region ap-southeast-2 --create-bucket-configuration LocationConstraint=ap-southeast-2
+{
+    "Location": "http://carineoliveira.s3.amazonaws.com/"
+}
+
+
+2. Upload the file in /classes/04class/aws_iam_storage/assets/index.html to the root of the S3 bucket.
+
+carineoliveira@Carines-MBP carineoliveira % aws s3 cp "${PWD}/../../../../../classes/04class/aws_iam_storage/assets/index.html"  s3://carineoliveira
+upload: ../../../aws_iam_storage/assets/index.html to s3://carineoliveira/index.html
+carineoliveira@Carines-MBP carineoliveira % 
+
+3. Enable the S3 website in the bucket
```

Comment:
Hi Carine,

Here you didn't create a website, you just shared a file publicly. It is possible to notice the difference in the url.
S3 websites have format `http://<BUCKET_NAME>.s3-website-<REGION>.amazonaws.com/`

Can you please check it?
https://docs.aws.amazon.com/cli/latest/reference/s3/website.html

Reply:
fixed